docs/how-to/load-models-from-pvc.md

```diff
@@ -0,0 +1,45 @@
+# Load Models from PVC
+
+You can store your models in a Persistent Volume Claim (PVC) and let KubeAI use them for serving.
+
+You must ensure the model files are already present in the PVC before creating the Model resource.
+
+Both vLLM and Ollama engines support loading models from PVCs.
+
+## vLLM
+
+For vLLM, use the following URL format:
+```yaml
+url: pvc://$PVC_NAME          # Loads the model from the PVC named $PVC_NAME
+url: pvc://$PVC_NAME/$PATH    # Loads from a specific path within the PVC
+```
+
+### PVC requirements
+
+vLLM supports both ReadWriteMany and ReadOnlyMany access modes. `Many` is used in order to support more than 1 vLLM replica.
+
+
+## Ollama
+
+For Ollama, use the following URL formats:
+```yaml
+url: pvc://$PVC_NAME?model=$MODEL_NAME    # Loads the model named $MODEL_NAME that's loaded on the disk
+url: pvc://$PVC_NAME/$PATH?model=$MODEL_NAME
+```
+
+### PVC Requirements
+Ollama requires using ReadWriteMany access mode because the rename operation `ollama cp` needs to write to the PVC.
```

Comment:
`because KubeAI needs to run ollama cp to alias the model name at startup` --> PS: There is probably a race condition here. Maybe we should make a note of that somewhere in the code.

Reply:
done